src/cmdstan-guide/pathfinder_config.Rmd

```diff
@@ -0,0 +1,162 @@
+# Pathfinder Method for Approximate Bayesian Inference
+
+The CmdStan method `pathfinder` uses the Pathfinder algorithm of @zhang_pathfinder:2022.
+Pathfinder is a variational method for approximately
+sampling from differentiable log densities.
+Starting from a random
+initialization, Pathfinder locates normal approximations to the target
+density along a quasi-Newton optimization path, with local covariance
+estimated using the inverse Hessian estimates produced by the
+L-BFGS optimizer.  Pathfinder returns draws from the Gaussian approximation
+with the lowest estimated Kullback-Leibler (KL) divergence to the true
+posterior.
```

Comment:
I would add "and optionally importance resampled from several runs of Pathfinder".